datalad/support/gitrepo.py

```diff
@@ -2344,7 +2344,10 @@ def get_submodules_(self, paths=None):
         Parameters
         ----------
         paths : list(pathlib.PurePath), optional
-            Restrict submodules to those under `paths`.
+            Restrict submodules to those under `paths`. Paths must be relative
+            to the resolved repository root, and must be normed to match the
+            reporting done by Git, i.e. no parent dir components
+            (ala "some/../this").
```

Comment:
I wonder if below for `Returns` we could clarify that paths below for submodules are Posix paths, or they aren't?

Reply:
I have no definitive answer. I took this description from the documentation. I did not plan to make changes to the reporting or parameterization.